Drop after x11/plasma6-libplasma >= 6.2.0 update
Drop after x11-wm/plasma6-kdecoration >= 6.2.0

CMake Warning at CMakeLists.txt:59 (find_package):
  Could not find a configuration file for package "Plasma" that is compatible
  with requested version "6.1".

  The following configuration files were considered but not accepted:

    /usr/local/lib/cmake/Plasma/PlasmaConfig.cmake, version: 6.1.2

CMake Error at CMakeLists.txt:69 (find_package):
  Could not find a configuration file for package "KDecoration2" that is
  compatible with requested version "6.2".

  The following configuration files were considered but not accepted:

    /usr/local/lib/cmake/KDecoration2/KDecoration2Config.cmake, version: 6.1.2

--- CMakeLists.txt.orig	2024-10-09 20:27:02 UTC
+++ CMakeLists.txt
@@ -57,7 +57,7 @@ set_package_properties(KF6Kirigami PROPERTIES
     TYPE RUNTIME
 )
 
-find_package(Plasma ${CMAKE_PROJECT_VERSION_MAJOR}.${CMAKE_PROJECT_VERSION_MINOR}
+find_package(Plasma ${CMAKE_PROJECT_VERSION_MAJOR}
   CONFIG
 )
 set_package_properties(Plasma PROPERTIES
@@ -66,7 +66,7 @@ set_package_properties(Plasma PROPERTIES
     TYPE RUNTIME
 )
 
-find_package(KDecoration2 ${CMAKE_PROJECT_VERSION_MAJOR}.${CMAKE_PROJECT_VERSION_MINOR}
+find_package(KDecoration2 ${CMAKE_PROJECT_VERSION_MAJOR}
   CONFIG REQUIRED
 )
 
